manticore/native/models.py

```diff
@@ -140,3 +140,38 @@ def strlen(state, s):
             ret = ITEBV(cpu.address_bit_size, byt == 0, offset, ret)
 
     return ret
+
+
+def strcpy(state, dst, src):
+    """
+    strcpy symbolic model
+
+    Algorithm: Copy every byte from the src to dst until finding a byte that will is or must be '\000'
+
+    :param State state: current program state
+    :param int dst: destination string address
+    :param int src: source string address
+    :return: pointer to the dst
+    :rtype: int
+    """
+    cpu = state.cpu
+    ret = dst
```

Comment:
I think this needs to be after we concretize `dst`, in the case that it's symbolic

Reply:
I'm also not sure if that matters, since we `raise` the exception anyway and divert control-flow. Idk.

Reply:
Ah I don't know either but rather be safe than sorry lol